addons/web/static/src/views/view_dialogs/select_create_dialog.js

```diff
@@ -0,0 +1,66 @@
+/** @odoo-module **/
+
+import { Dialog } from "@web/core/dialog/dialog";
+import { View } from "../view";
+import { useService } from "@web/core/utils/hooks";
+
+const { onWillStart, useState, xml } = owl;
+
+export class SelectCreateDialog extends Dialog {
+    setup() {
+        super.setup();
+        this.viewService = useService("view");
+        this.state = useState({
+            selectedRecords: [],
+        });
+
+        this.propsView = {
+            resModel: this.props.resModel,
+            domain: this.props.domain,
+            context: this.props.context,
+            type: "list",
+            editable: false,
+            showButtons: false,
+            hasSelectors: !this.props.disableMultipleSelection,
```

Comment:
I'd rename this props to avoid double negation -> `!disableMultipleSelection === allowMultipleSelection`, or shorter `allowMultiSelect` or even `multiSelect`